api/v1beta1/kafkacluster_types.go

```diff
@@ -134,15 +153,25 @@ const (
 
 // KafkaClusterSpec defines the desired state of KafkaCluster
 type KafkaClusterSpec struct {
+	// kRaft is used to decide where the Kafka cluster is under KRaft mode or ZooKeeper mode.
+	// This is default to be true; if set to false, the Kafka cluster is in ZooKeeper mode.
+	// +kubebuilder:default=true
```

Comment:
```suggestion
	// +kubebuilder:default=false
```
I think zk is still the default approach and make more sense to have this set default to false.